src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,121 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './FindMovie.scss';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import classNames from 'classnames';
+import { MovieCard } from '../MovieCard';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+type Props = {
+  addMovie: (newMovie: Movie) => void
+};
+
+export const FindMovie: React.FC<Props> = (props) => {
+  const [movieTitle, setMovieTitle] = useState('');
+  const [isMovieFound, showMovie] = useState(true);
+  const [movie, setMovie] = useState<Movie | null>(null);
+
+  useEffect(() => {
+    showMovie(true);
+  }, [movieTitle]);
+
+  const handleMovie = async () => {
+    const response = await fetch(`https://www.omdbapi.com/?apikey=51c89ae5&t=${movieTitle}`);
+
+    const movieFromServer = await response.json();
+
+    if (movieFromServer.Response === 'False') {
+      showMovie(false);
+
+      return null;
+    }
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+    showMovie(true);
+
+    return {
+      Poster: movieFromServer.Poster,
+      Title: movieFromServer.Title,
+      Plot: movieFromServer.Plot,
+      imdbID: movieFromServer.imdbID,
+    };
```

Comment:
if I try to find another movie after I already found one, newMovie should be null, at this moment it's possible to add a movie from the previous response if the current one is false, fix this